src/components/Files/FilesTab.tsx

```diff
@@ -601,19 +602,35 @@ const FileUploadDialog = ({
                   <CareIcon icon="l-times" />
                 </Button>
               </div>
-              <TextFormField
-                name={`file_name_${index}`}
-                type="text"
-                label={t("enter_file_name")}
-                id={`upload-file-name-${index}`}
-                required
-                value={fileUpload.fileNames[index] || ""}
-                disabled={fileUpload.uploading}
-                onChange={(e) => fileUpload.setFileName(e.value, index)}
-                error={
-                  index === 0 && fileUpload.error ? fileUpload.error : undefined
-                }
-              />
+              <div className="space-y-2">
+                <Label
+                  htmlFor={`upload-file-name-${index}`}
+                  className="text-sm font-medium text-gray-700"
+                >
+                  {t("enter_file_name")}
+                </Label>
+                <Input
+                  name={`file_name_${index}`}
+                  type="text"
+                  id={`upload-file-name-${index}`}
+                  required
+                  className={`block w-full rounded-md border px-3 py-2 text-sm ${
+                    index === 0 && fileUpload.error
+                      ? "border-red-500 focus:ring-red-500"
+                      : "border-gray-300 focus:ring-primary"
+                  }`}
```

Comment:
let's not deviate away from configured styles
```suggestion
                  className={
                    index === 0 && fileUpload.error
                      ? "border-red-500 focus:ring-red-500"
                      : "border-gray-300 focus:ring-primary"
                 }
```